engine/jni/src/wscript

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+
+def init(ctx):
+    pass
+
+def options(opt):
+    opt.load('waf_dynamo')
+
+def configure(conf):
+    conf.recurse('test')
+
+def build(bld):
+    bld.stlib(features = 'cxx',
+               includes = ['.'],
+               source   = bld.path.ant_glob('jni/*.cpp'),
+               use      = 'JDK',
+               target   = 'jni')
+
+    bld.stlib(features = 'cxx skip_asan',
+               includes = ['.'],
+               source   = bld.path.ant_glob('jni/*.cpp'),
+               use      = 'JDK',
+               target   = 'jni_noasan')
```

Comment:
Our java plugins still don't like it when we use asan et al (it cannot figure out where the shared libraries should be loaded from)